opensearch/connection/base.py

```diff
@@ -127,6 +131,29 @@ def __init__(
         if opaque_id:
             self.headers["x-opaque-id"] = opaque_id
 
+        if (
+            os.getenv("ELASTIC_CLIENT_APIVERSIONING") == "1"
```

Comment:
Consider to use local variables for results of `getenv`: they are used several times across the code.

Reply:
Thanks! Parking this PR as of now. Support for Elasticsearch header was added as a part of #58 , and this PR will need to be modified to add OpenSearch headers, if we decide to add them